Tools/CISupport/ews-build/steps.py

```diff
@@ -2181,9 +2180,10 @@ def send_email_for_new_build_failure(self):
                 change_author, errors = GitHub.email_for_owners(self.getProperty('owners', []))
                 for error in errors:
                     print(error)
+                    self._addToLog('stdio', error)
 
             if not change_author:
-                print('Failed to determine change author for hash {} belonging to PR #{}'.format(sha, patch_id))
+                self._addToLog('Failed to determine change author for hash {} belonging to PR #{}'.format(sha, patch_id))
```

Comment:
Might want to mention something about contributors.json as well. So that people know what they can do to fix it. Might want to include github username in this message as well. Something like: 
`Unable to determine email address for [github_username] from contributors.json. Skipping sending email.`